doc/logo/logo_yaml2sbml_icon.svg

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="44.38686mm"
+   height="42.568233mm"
+   viewBox="0 0 44.386857 42.568233"
+   version="1.1"
+   id="svg8"
+   inkscape:version="1.0.1 (0767f8302a, 2020-10-17)"
+   sodipodi:docname="logo_yaml2sbml_icon.svg"
+   inkscape:export-filename="/home/yannik/yaml2sbml/doc/logo/logo_yaml2sbml_icon.svg.png"
```

Comment:
Not sure if this matters functionally, or if you care about this @yannikschaelte

Reply:
Also for some of the other SVG files

Reply:
Should not really matter. Ideally, one would hope that inkscape only uses relative pathnames, but should be fine.